metrics/lib/common.bash

```diff
@@ -60,3 +60,27 @@ function check_cmds()
 		echo "command: $cmd: yes"
 	done
 }
+
+# Initialization/verification environment. This function makes
+# minimal steps for metrics/tests execution.
+function init_env()
+{
+	cmd=("docker")
+	contr_running=$(docker ps -qa)
+
+	check_cmds "${cmd[@]}"
+
+	# Verify containers running
+	if [ ! -z "$contr_running" ];then
+		docker rm -f $(docker ps -qa)
+	fi
+
+	# Restart services
```

Comment:
I think it is necessary in order to get more reliable results, however as you said, it is better to describe as a "WARNING" in documentation.